test/fixtures/polygon/boxLocation.js

```diff
@@ -0,0 +1,78 @@
+module.exports = {
+  tolerance: 0.0075,
+  config: {
+    type: 'scatter',
+    options: {
+      scales: {
+        x: {
+          display: true,
+          min: -10,
+          max: 10
+        },
+        y: {
+          display: true,
+          min: -10,
+          max: 10
+        }
+      },
+      plugins: {
+        legend: false,
+        annotation: {
+          drawTime: 'afterDraw',
+          annotations: {
+            triangle1: {
```

Comment:
I'm sure there is a good explanation, but why aren't the triangles centered vertically to the boxes?

Reply:
let me do another test before answering just to be sure about what I'll write

Reply:
After giving it thought, it seems obvious the center of the points is vertically centered, not the outer bounds of the triangle that I was looking at.

Reply:
I tested which is confirming my answer. The triangle is built using the center of the box and the circle built inside the box with radius (in this case), in this case the min between height and width of the box (because radius is NaN).

Here the result drawing the circle too:

![boxes](https://user-images.githubusercontent.com/11741250/144761391-2c61df68-3625-4a54-b462-149868d82782.png)

I have changed the test case in order to highlight also the circle.

Reply:
Good, the added circle clarifies it nicely without having to thing about it :)